kernel-open/nvidia-drm/nvidia-drm-drv.c

```diff
@@ -826,7 +826,9 @@ static struct drm_driver nv_drm_driver = {
     .gem_prime_get_sg_table = nv_drm_gem_prime_get_sg_table,
     .gem_prime_vmap         = nv_drm_gem_prime_vmap,
     .gem_prime_vunmap       = nv_drm_gem_prime_vunmap,
-
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 0, 0)
```

Comment:
I know this still says WIP, but take a look at this related conftest instead of kernel version check:
https://github.com/NVIDIA/open-gpu-kernel-modules/blob/main/kernel-open/conftest.sh#L4342-L4364